pkgs/development/tools/language-servers/csharp-ls/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib
+, buildDotnetGlobalTool
+, dotnetCorePackages
+}:
+let
+  inherit (dotnetCorePackages) sdk_7_0;
+in
+
+buildDotnetGlobalTool {
+  pname = "csharp-ls";
+  version = "0.8.0";
+
+  nugetSha256 = "sha256-F3N6ESE/VmQA5sOMm3eqSkhSNVCCsnAXTCC+McsAnQw=";
+
+  dotnet-sdk = sdk_7_0;
+  dotnet-runtime = sdk_7_0;
+
+  meta = with lib; {
+    description = "Roslyn-based LSP language server for C#";
+    homepage = "https://github.com/razzmatazz/csharp-language-server";
+    changelog = "https://github.com/razzmatazz/csharp-language-server/releases/tag/v${version}";
```

Comment:
Does `version` here mean `lib.version`?

Reply:
No, it should be the package version. I have seen this way of specifying `meta.changelog` in various other packages such as: https://github.com/NixOS/nixpkgs/blob/3cee93dfaf3b5e9e9db719022fdb271c2f399aae/pkgs/games/freenukum/default.nix#L69

Reply:
Yeah, but that requires the `rec` keyword, right?

Reply:
Oh maybe ! I added it :)